mpm88_desktop/linux_compile_and_run.sh

```diff
@@ -1,9 +1,12 @@
-BACKEND_NAME="cuda" # cuda, x64, vulkan
-TAICHI_REPO="/home/taichigraphics/workspace/taichi"
+BACKEND_NAME="vulkan" # cuda, x64, vulkan
+TAICHI_REPO="/home/taichigraphics/workspace/taichi2"
```

Comment:
nit: we might actually want to leave this blank when merging this script since it's different in everyone's desktop?

Reply:
Sure~